operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -13,7 +13,7 @@ include::modules/configure-ocp-oauth-identity-provider.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-* If you use a custom TLS certificate for {ocp} OAuth server, you must add the CA’s root certificate to {product-title} as a trusted root CA. Otherwise, Central cannot connect to the {ocp} OAuth server.
+* If you use a custom TLS certificate for {ocp} OAuth server, you must add the CA's root certificate to {product-title} as a trusted root CA. Otherwise, Central cannot connect to the {ocp} OAuth server.
```

Comment:
To facilitate localisation, you could update "the CA's root certificate" to "the root certificate of the CA", to avoid using a possessive.